web/src/pages/your_account.rs

```diff
@@ -0,0 +1,299 @@
+            get_users_agent,
+            pubkey_to_username: HashMap::new(),
+            refresher: Interval::new(1000, move || {
+                refresh.emit(());
+            }),
+        }
+    }
+
+    fn update(&mut self, message: Message) -> ShouldRender {
+        match message {
+            Message::GotBalance(res) => {
+                match res {
+                    Ok(balance) => self.user_balance = Some(balance),
+                    Err(_) => self
+                        .get_balance_agent
+                        .send(self.props.user.0.clone().to_thin()),
+                }
+                true
+            }
+
+            Message::SelectUser(user) => {
+                self.user_to_send_to = Some(user);
+                false
+            }
+            Message::UpdateAmount(amount) => {
+                self.amount_to_send = amount;
+                true
+            }
+            Message::SendTransaction => {
+                if let Some(amount) = validate_amount(&self.user_balance, &self.amount_to_send) {
+                    if let Some(user_to_send_to) = self.user_to_send_to.clone() {
+                        let sequence = self.props.user.1 + 1;
+
+                        self.send_asset_agent.send((
+                            self.props.user.0.clone(),
+                            sequence,
+                            user_to_send_to,
+                            amount,
+                        ));
+
+                        self.props.bump_sequence.emit(sequence);
+                    }
+                }
+
+                // TODO in background
+                self.get_balance_agent
+                    .send(self.props.user.0.clone().to_thin());
+
+                false
+            }
+            Message::TransactionSent(ret) => {
+                ret.unwrap(); // TODO send asset in dialog
+                false
+            }
+
+            Message::LatestTransactionsGot(mut latest_transactions) => {
+                latest_transactions.reverse();
+                self.latest_transactions = latest_transactions;
+
+                self.get_balance_agent
+                    .send(self.props.user.0.clone().to_thin());
+
+                true
+            }
+            Message::GotUsers(users) => {
+                self.pubkey_to_username = users
+                    .into_iter()
+                    .map(|user| (*user.public_key(), user.name.clone()))
+                    .collect();
+
+                true
+            }
+
+            Message::Refresh => true,
+        }
+    }
+
+    fn change(&mut self, props: Properties) -> ShouldRender {
+        self.props = props;
+        false
+    }
+
+    fn view(&self) -> Html {
+        let now = Utc::now();
+
+        html! { <>
+            <h1> { "Your account" } </h1>
+
+            <p> { "
+                Now, your account is registered on the chain.
+                Same as Bitcoin, you have a wallet, to which we added 100'000
+                assets.
```

Comment:
Still need a better name for this.

Reply:
So it seems we're using `assets` now. Which is fine.

Reply:
The icon directly?

Reply:
Trying out things: `100'000:icon: assets`?

Reply:
Going for: 100'000 ₳ (assets)